modules/options/static.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 The Gitea Authors. All rights reserved.
+// Copyright 2016-2022 The Gitea Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2016 The Gitea Authors. All rights reserved.
```

Reply:
How did we handle it when updating existing files?
Did we really not update the copyright?
Or did we simply replace it with the current year?
```suggestion
// Copyright 2022 The Gitea Authors. All rights reserved.
```